### PR TITLE
simplify README content, 1st pass, re #1148

### PR DESCRIPTION
**Pull request by ghost**
(further passes needed, though you could merge this already)

### CODE REVIEW

**Review by Honglei-Cong — DISMISSED** [review]

**Comment by Honglei-Cong** [comment]
@laizy

File: README.md [hunk 1]
```diff
@@ -106,41 +79,21 @@ The Ontology CLI can run nodes for the MainNet, TestNet and local PrivateNet. Ch
 
 You can run an Ontology MainNet node built from the source code with:
 
- ``` shell
+```shell
 ./ontology
```
Comment (laizy) [inline README.md]:
```suggestion
./ontology                                      # Linux
```

File: README.md [hunk 2]
```diff
@@ -106,41 +79,21 @@ The Ontology CLI can run nodes for the MainNet, TestNet and local PrivateNet. Ch
...
- ./ontology-darwin-amd64 --networkid 2
- ```
-
- To run it with a Windows release build:
-
- ``` shell
- start ontology-windows-amd64.exe --networkid 2
- ```
+```shell
+./ontology --networkid 2
```
Comment (laizy) [inline README.md]:
```suggestion
./ontology --networkid 2                                    # Linux
```

File: README.md [hunk 3]
```diff
@@ -152,8 +105,10 @@ The Ontology CLI allows you to run a local PrivateNet on your computer. Before y
 
 To start the PrivateNet built from the source code with:
 
-``` shell
+```shell
 ./ontology --testmode
```
Comment (laizy) [inline README.md]:
```suggestion
./ontology --testmode                                   # Linux
```

File: README.md [hunk 4]
```diff
@@ -152,8 +105,10 @@ The Ontology CLI allows you to run a local PrivateNet on your computer. Before y
...
+./ontology-darwin-amd64 --testmode          # MacOSs
```
Comment (laizy) [inline README.md]:
```suggestion
./ontology-darwin-amd64 --testmode          # MacOS
```

File: README.md [hunk 5]
```diff
@@ -80,7 +80,7 @@ The Ontology CLI can run nodes for the MainNet, TestNet and local PrivateNet. Ch
 You can run an Ontology MainNet node built from the source code with:
 
 ```shell
-./ontology
+./ontology                                      # Linux
```
Comment (Honglei-Cong) [inline README.md]: `Linux` is not aligned with the following `MacOS`

**Review by laizy — APPROVED** [review]
Thanks.

**Pull request merged**